rke/templates/templates.go

```diff
@@ -99,12 +100,19 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.16.0-alpha": weavev18,
 		},
 		kdm.NginxIngress: {
-			">=1.8.0-rancher0 <1.13.10-rancher1-3":  nginxIngressv18,
-			">=1.13.10-rancher1-3 <1.14.0-rancher0": nginxIngressV115,
-			">=1.14.0-rancher0 <=1.14.6-rancher1-1": nginxIngressv18,
-			">=1.14.6-rancher2 <1.15.0-rancher0":    nginxIngressV115,
-			">=1.15.0-rancher0 <=1.15.3-rancher1-1": nginxIngressv18,
-			">=1.15.3-rancher2":                     nginxIngressV115,
+			">=1.8.0-rancher0 <1.13.10-rancher1-3":   nginxIngressv18,
+			">=1.13.10-rancher1-3 <1.14.0-rancher0":  nginxIngressV115,
+			">=1.14.0-rancher0 <=1.14.6-rancher1-1":  nginxIngressv18,
+			">=1.14.6-rancher2 <1.15.0-rancher0":     nginxIngressV115,
+			">=1.15.0-rancher0 <=1.15.3-rancher1-1":  nginxIngressv18,
+			">=1.15.3-rancher2 <1.15.12-rancher1-2":  nginxIngressV115,
+			"=1.15.12-rancher1-2":                    nginxIngressV11512,
+			">1.15.12-rancher1-2 <1.16.9-rancher1-2": nginxIngressV115,
+			"=1.16.9-rancher1-2":                     nginxIngressV11512,
+			">1.16.9-rancher1-2 <1.17.5-rancher1-2":  nginxIngressV115,
+			"=1.17.5-rancher1-2":                     nginxIngressV11512,
+			">1.17.5-rancher1-2 <1.18.2-rancher1-2":  nginxIngressV115,
```

Comment:
This can just be `>1.17.5-rancher1-2` as any 1.18.x template will have the latest nginx version

Reply:
`>=1.17.5-rancher1-2 -> nginxIngressV11512`